build.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+clear
```

Comment:
It's considered "rude" for a script clear the terminal scrollback buffer like that.

```suggestion
```